styleguide/_static/issue-templates/bug.md

```diff
@@ -0,0 +1,76 @@
+---
+name: 🐞 Bug Report 
+about: File a bug report 
+title: 🐞 <Insert Title> 
+labels: bug
+assignees: ''
+
+---
+
+### Checklist
+In order to improve the time, how long a bugfix does take, please make sure you address as much as possible of the following checklist.
+
+- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.
+- [ ] I have reproduced the issue with at least the latest released version of \<PROJECT-NAME\>.
```

Comment:
Hm, maybe I understand something wrong, but actually that is something I would not expect in a style guide for a programming language.....

Reply:
Well yes and no, it definitively is not language specific problem/guide, but also we don't have anything like it yet. So my thinking was we can develop it as part of this guide and move it to a "more" general one later on (if needed/wanted).